swe_checkstyle.xml

```diff
@@ -9,27 +9,26 @@
 <!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.2//EN" "http://www.puppycrawl.com/dtds/configuration_1_2.dtd">
 <module name="Checker">
     <property name="severity" value="error"/>
-    <module name="FileLength"/>
```

Comment:
There are a lot of changes in this file that I don't 
understand why.
Is it newline changes or something?

Reply:
could you review the checkstyle changes in this PR please?
https://github.com/jenkinsci/build-failure-analyzer-plugin/pull/98

checkstyle had to be upgraded so that java 8 code would work.

Reply:
merged that